### DISCUSSION TITLE
"Copy as grpcurl" for gRPC requests

### DISCUSSION THREAD

**Discussion by ravanscafi** [Kong/insomnia#7180] [Ideas] [answered]
For regular HTTP requests, we can copy the requests as cURL:
<img width="155" alt="image" src="https://github.com/Kong/insomnia/assets/6104262/16a54eec-5742-43c1-a151-55e91c35975f">

However, there's no similar way to do that with gRPC requests. 
<img width="107" alt="image" src="https://github.com/Kong/insomnia/assets/6104262/b1437e6b-4ae9-4d41-8165-012b7676b6b7">

[grpcurl](https://github.com/fullstorydev/grpcurl) is the most used cURL-like tool to interact with gRPC requests and it would be awesome to have such ability.

Thanks!

**Comment by hughgardiner** [comment]
this would be very helpful!

**Comment by poojithap11** [comment, marked as answer]
Hi,

Thank you for your suggestion.

A feature request has been raised to add a "Copy as grpcurl" option for gRPC requests in insomnia. 

Thank you.
